src/go/rpk/pkg/cli/cmd/cluster/config/set.go

```diff
@@ -0,0 +1,90 @@
+// Copyright 2022 Vectorized, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package config
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/config"
+	"github.com/vectorizedio/redpanda/src/go/rpk/pkg/out"
+	"gopkg.in/yaml.v3"
+)
+
+func newSetCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "set <key> <value>",
+		Short: "Set a single cluster configuration property",
+		Long: `Set a single cluster configuration property.
+
+This command is provided for use in scripts.  For interactive editing, or bulk
+changes, use the 'edit' and 'import' commands respectively.`,
+		Run: func(cmd *cobra.Command, args []string) {
+			if len(args) != 2 {
+				out.Die("Usage: set <key> <value>")
+			}
```

Comment:
Potentially worth going flags for multiple key/value support, but if not, this can be `Args: cobra.ExactArgs(2)`.